R/aaa.R

```diff
@@ -21,6 +21,7 @@ format_root_section <- function() {
 
 format_root_criteria_items <- function(indent = 0) {
   format <- format(.root_env$root$crit)[-1L]
+  format <- gsub("'([^']+)'", "`\\1`", format)
```

Comment:
New location for this fixup and now looking for single-quoted strings instead of double-quoted.